xtas/tasks/single.py

```diff
@@ -319,8 +319,8 @@ def semanticize(doc, lang='en'):
         raise ValueError("not a valid language: %r" % lang)
     text = fetch(doc)
     url = 'http://semanticize.uva.nl/api/%s?%s' % (lang,
-                                                   urlencode({'text': text}))
-    return json.loads(urlopen(url).read())['links']
```

Comment:
Can the server deal with lots of people running xtas on clusters, with many parallel tasks? If not do we need some kind of throttling or error handling?